elements/map/stories/coordinates-custom-tooltips.js

```diff
@@ -0,0 +1,148 @@
+    <b>LON:</b> ${newFeature.get("lon")}
+  <br>
+    <i class="small"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><title>earth</title><path d="M17.9,17.39C17.64,16.59 16.89,16 16,16H15V13A1,1 0 0,0 14,12H8V10H10A1,1 0 0,0 11,9V7H13A2,2 0 0,0 15,5V4.59C17.93,5.77 20,8.64 20,12C20,14.08 19.2,15.97 17.9,17.39M11,19.93C7.05,19.44 4,16.08 4,12C4,11.38 4.08,10.78 4.21,10.21L9,15V16A2,2 0 0,0 11,18M12,2A10,10 0 0,0 2,12A10,10 0 0,0 12,22A10,10 0 0,0 22,12A10,10 0 0,0 12,2Z" /></svg></i>
+    <b>LAT:</b> ${newFeature.get("lat")}
+
+
+</div>
+`;
+    }
+  },
+);
+
+/// Utility function to normalize values for color rendering
+const normalize = (value, max = 1) => ["/", value, max];
+
+const falseRed = normalize(["band", 1], 4462);
+const falseGreen = normalize(["band", 2], 2820);
+const falseNir = normalize(["band", 4], 6214);
+// Utility function to create a color array for false color rendering style
+const falseColor = {
+  color: [
+    "case",
+    [
+      "any",
+      ["==", ["band", 1], 0],
+      ["==", ["band", 2], 0],
+      ["==", ["band", 3], 0],
+      ["==", ["band", 4], 0],
+    ],
+    "#00000000",
+    ["array", falseNir, falseRed, falseGreen, 1],
+  ],
+};
+
+/**
+ * Renders `eox-map` with `Tooltip` on click over a feature
+ *
+ * @returns {Object} The story configuration with arguments for the component.
+ */
+const CoordinatesCustomTooltipsStory = {
+  args: {
+    layers: [
+      {
+        type: "WebGLTile",
+        properties: {
+          id: "multipleCOGs",
+        },
+        style: falseColor,
+        source: {
+          type: "GeoTIFF",
+          normalize: false,
+          sources: [
+            {
+              url: "https://sentinel-cogs.s3.us-west-2.amazonaws.com/sentinel-s2-l2a-cogs/36/Q/WD/2020/7/S2A_36QWD_20200701_0_L2A/B04.tif",
+              max: 4462,
+            },
+            {
+              url: "https://sentinel-cogs.s3.us-west-2.amazonaws.com/sentinel-s2-l2a-cogs/36/Q/WD/2020/7/S2A_36QWD_20200701_0_L2A/B03.tif",
+              max: 2820,
+            },
+            {
+              url: "https://sentinel-cogs.s3.us-west-2.amazonaws.com/sentinel-s2-l2a-cogs/36/Q/WD/2020/7/S2A_36QWD_20200701_0_L2A/B02.tif",
+              max: 2170,
+            },
+            {
+              url: "https://sentinel-cogs.s3.us-west-2.amazonaws.com/sentinel-s2-l2a-cogs/36/Q/WD/2020/7/S2A_36QWD_20200701_0_L2A/B08.tif",
+              max: 6214,
+            },
+          ],
+        },
+        interactions: [
+          {
+            type: "select",
+            options: {
+              id: "selectInteraction",
+              condition: "pointermove",
+              precision: 2,
+              coordinates: true,
```

Comment:
Could we show the coordinates in a separate example which is not WebGLTile? In order to make it more findable in the story list; ideally:
- one story for coordinates
- one story for WMS feature info
- one story for WebGLTile data extraction

Reply:
since coordinates at the current implementation are provided with the selct layers ( vector/WebGL/WMS ). 
Probably having it within the WebGLTile layer story now would be okay. 
I'll create a separate issue for allowing coordinates for all othe types of layers.